packages/bpk-docs/src/pages/ModalsPage/ModalsPage.js

```diff
@@ -146,6 +146,14 @@ const ModalsPage = () => (
     ]}
     components={components}
     readme={modalReadme}
+    usageTable={{
+      dos: ['Use for rich / complex content related to the current page.'],
+      donts: [
+        "Don't use for alerts, errors and confirmations - use a dialog instead.",
+        `Don't overuse modals as interrupts as they will get in the way of the
+      user's main tasks, and may disrupt context.`,
```

Comment:
This sentence is grammatically incorrect. Perhaps: "Don't overuse modals. They are an intrusive UI pattern and should be used sparingly." Im not sure though...

Reply:
I heard "grammar" and came as fast as I could.

I think it's probably grammatically correct, but it's a bit ambiguous as we're using 'interrupt' as a noun. I like what Matt suggests as it's less ambiguous.

Reply:
Fixed.